_maps/map_files/MetaStation/MetaStation.dmm

```diff
@@ -50100,7 +50105,7 @@
 "csa" = (
 /obj/structure/table,
 /obj/item/restraints/handcuffs/cable/white,
-/obj/item/gun/syringe,
```

Comment:
what I said for the maint EMP grenade also applies to the maint syringe gun, I think.